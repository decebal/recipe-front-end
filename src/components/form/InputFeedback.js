import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const StyledFeedbackInput = styled.label`
  color: #999;
  margin-top: .25rem;
`;

const InputFeedback = ({ error }) => (
  error
    ? <StyledFeedbackInput>{error}</StyledFeedbackInput>
    : null
);

InputFeedback.propTypes = {
  error: PropTypes.string,
};

InputFeedback.defaultProps = {
  error: null,
};

export default InputFeedback;
