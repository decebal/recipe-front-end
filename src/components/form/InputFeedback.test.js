import React from 'react';
import renderer from 'react-test-renderer';
import InputFeedback from './InputFeedback';


test('InputFeedback renders correctly', () => {
  const tree = renderer
    .create(<InputFeedback />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
