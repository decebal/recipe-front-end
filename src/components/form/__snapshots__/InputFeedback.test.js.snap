// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputFeedback renders correctly 1`] = `null`;
